vite.config.js

```diff
@@ -27,6 +28,13 @@ export default ({mode}) => {
             manifest: isOpen ? 'manifest.open.json' : 'manifest.json',
             outDir: `public/${buildDir}`,
         },
+        resolve: {
+            alias: {
+                '@': path.resolve(import.meta.dirname, './resources/js/'),
+                '%store': path.resolve(import.meta.dirname, './resources/js/bootstrap/store.js'),
+                '%router': path.resolve(import.meta.dirname, './resources/js/bootstrap/router.js'),
+            },
+        },
```

Comment:
only works in node 21.x+

Reply:
Fixed in bedae20ad5a0bbcaf93a848be68df024f278fbee